src/renderer/entities/wallet/ui/Cards/WalletCardLg.tsx

```diff
@@ -5,7 +5,7 @@ import { FootnoteText, StatusLabel } from '@shared/ui';
 import { cnTw } from '@shared/lib/utils';
 import type { Wallet } from '@shared/core';
 import { WalletIcon } from '../WalletIcon/WalletIcon';
-import { walletUtils } from '../../lib/wallet-utils';
+import { walletUtils } from '../..';
```

Comment:
Let's not go though `index.ts`
Import looks strange and I'm afraid of getting errors in future